### PR TITLE
Add R mrgsolve

### PR DESCRIPTION
**Pull request by apeltzer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-mrgsolve/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    - r-bh >=1.62.0_1
+    - r-rcpp >=0.12.12
+    - r-rcpparmadillo >=0.7.900.2.0
+    - r-dplyr >=0.8.1
+    - r-glue
+    - r-lifecycle
+    - r-magrittr >=1.5
+    - r-rlang >=0.3.4
+    - r-tibble >=2.1.1
+    - r-tidyselect >=0.2.5
```
Comment (dbast) [inline recipes/r-mrgsolve/meta.yaml:45]:
```suggestion
    - r-tidyselect >=0.2.5
    - libblas
```

File: recipes/r-mrgsolve/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    - libblas  # [unix]
```
Comment (dbast) [inline recipes/r-mrgsolve/meta.yaml:46]:
```suggestion
    - libblas  # [unix]
    - liblapack  # [unix]
```

**Comment by apeltzer** [comment]
@conda-forge-admin, please add bot automerge

**Comment by apeltzer** [comment]
@conda-forge/help-r - ready for review i think

**Comment by apeltzer** [comment]
And thanks @dbast for fixing the last bit and pointing me to Björns script 👍🏻

**Pull request merged**